src/resources/AmalgamationRegular/BC.ts

```diff
@@ -32,39 +32,6 @@ export const AmalgamationRegResourceBc: AmalgamationRegResourceIF = {
   shareClasses: {
     countMinimum: 1
   },
-  incorporationArticles: {
-    articles: '<br>BC LTD Articles.', // Line break is required to prevent tooltip from splitting the screen.
-    articlesTooltip: 'Articles should outline the rules and procedures for corporate matters such as holding ' +
-    'meetings, issuing and transferring shares, and duties of directors and officers.'
-  },
-  incorporationAgreement: {
-    helpSection: [
-      {
-        header: 'What is the sample Incorporation Agreement and Company Articles?',
-        helpText: [
-          `The sample Incorporation Agreement and Company Articles is a template that you can use to create an 
-            incorporation agreement and articles for your company. It uses all the standard provisions by legislation. 
-            There are three types of samples depending on if you're incorporating a Limited Company, a Benefit Company, 
-            or a Community Contribution Company.`,
-          `If you would like to customize any other provisions in the Articles, you cannot use these samples. 
-            We recommend seeking professional assistance from a lawyer or accountant to help you prepare your articles.`
-        ]
-      }
-    ],
-    article: 'bc_limited_company_incorporation_agreement.pdf',
-    documents: [
-      {
-        code: 'sample',
-        description: 'The <b>sample Incorporation Agreement and Articles</b> ' +
-          'has been completed and a copy has been added to the company\'s record book.'
-      },
-      {
-        code: 'custom',
-        description: 'The <b>custom Incorporation Agreement and custom Articles</b> ' +
-          'has been completed and a copy has been added to the company\'s record book.'
-      }
-    ]
-  },
```

Comment:
This data was not used for regular amalgamations. (It was carried over from the initial resources files, which were copies of the IA resources.)